parsers/JP.py

```diff
@@ -70,9 +70,6 @@ def fetch_production_df(zone_key='JP-TK', session=None, target_datetime=None,
     Calculates production from consumption and imports for a given area
     All production is mapped to unknown
     """
-    if target_datetime:
```

Comment:
Is the parser now able to parse historical data?

Reply:
I have just checked that they can parse up to 2020-05-01 except the Hokkaido one which is excluded now.